tests/test_environ.py

```diff
@@ -162,17 +165,17 @@ def test_load_static_config(s, exp, loaded, tmpdir, xonsh_builtins):
 
 
 @skip_if_on_unix
-def test_locate_binary_on_windows():
+def test_locate_binary_on_windows(xonsh_builtins):
     files = ('file1.exe', 'FILE2.BAT', 'file3.txt')
     with TemporaryDirectory() as tmpdir:
         for fname in files:
             fpath = os.path.join(tmpdir, fname)
             with open(fpath, 'w') as f:
                 f.write(fpath)
         env = Env({'PATH': [tmpdir], 'PATHEXT': ['.COM', '.EXE', '.BAT']})
```

Comment:
Could `xonsh_builtins.__xonsh_env__` not be an empty instance of the `Env` class by default (i.e. define in the fixture. Then we could just do the following here: 

```
xonsh_builtins.__xonsh_env__.update({
    'PATH': [tmpdir],
    'PATHEXT': ['.COM', '.EXE', '.BAT'],
})
```